.prettierignore

```diff
@@ -43,6 +43,9 @@ packages/website/versioned_docs/**/reference/
 packages/website/versioned_docs/**/standard-library/built-in-decorators.md
 packages/website/versioned_docs/**/standard-library/built-in-data-types.md
 
+# Docs with special formatting that prettier can't handle
+docs/getting-started/getting-started-rest/
```

Comment:
can you explain more what is failing here? this is the first time i have seen this

Reply:
Sure, my user feedback indicated it was difficult to tell where changes were made in the example code.

There's not a way to highlight specific lines within a code block using markdown, so I added special comments in the code examples to indicate where the changes are made, specifically `// <+>` at the end of each new line. 

Prettier was consistently moving those comments to the following line, rendering them useless for the intended purpose.

Reply:
you can use that to highlight lines in docusaurus markdown https://docusaurus.io/docs/markdown-features/code-blocks#highlighting-with-metadata-string

Reply:
Awesome! I'll make changes to use this syntax and resubmit.

Reply:
depending on which one you prefer you might have to use the 2nd one(with the comment) and add that config in docusaurus.config.ts as we never did it before